Core/ELF/ElfReader.cpp

```diff
@@ -368,7 +365,7 @@ void ElfReader::LoadRelocations2(int rel_seg)
 				break;
 			case 1:
 			case 5: // R_MIPS_LO16
-				op = (op&0xffff0000) | (((op&0xffff)+relocate_to)&0xffff);
+				op = (op & 0xffff0000) | ((static_cast<int>(short(op)) + relocate_to) & 0xffff);
```

Comment:
I don't think this is actually correct because R_MIPS_HI16 accounts for it?

-[Unknown]